rasa/src/Rasa/Internal/Buffer.hs

```diff
@@ -50,6 +51,9 @@ instance HasBuffer Buffer where
 text :: HasBuffer b => Lens' b Y.YiString
 text = buffer.text'
 
+getText :: HasBuffer b => Getter b Y.YiString
```

Comment:
You can move the previously mentioned docstring right here and Haddock will still pick it up 😋